src/components/FeeSelector/index.tsx

```diff
@@ -123,26 +144,58 @@ function FeeSelector({
   currencyA: Currency | undefined
   currencyB: Currency | undefined
 }) {
+  const theme = useTheme()
   const [show, setShow] = useState(false)
   const feeTierDistribution = useFeeTierDistribution(currencyA, currencyB)
 
+  const { data: farms } = useProMMFarms()
+  const getProMMFarms = useGetProMMFarms()
+  const blockNumber = useBlockNumber()
```

Comment:
use this `useProMMFarmsFetchOnlyOne` to avoid get farm info again, we dont need update farm data here

Reply:
Fixed